source/switching_to_live/set_up_a_live_worldpay_account/index.html.md.erb

```diff
@@ -1,187 +1,100 @@
-
-__CANCELLED__
-
-__CAPTURED__
-
-__CAPTURE_FAILED__
-
-__SETTLED__
-
-__SETTLED_BY_MERCHANT__
-
-__SENT_FOR_REFUND__
-
-__REFUNDED__
-
-__REFUNDED_BY_MERCHANT__
-
-__REFUSED__
-
-__REFUSED_BY_BANK__
-
-__REFUND_FAILED__
-
-## Test your configuration
-
-
-1. Sign into the GOV.UK Pay admin tool and confirm that the [payment cards](https://selfservice.payments.service.gov.uk/card-types/summary) you want to accept are enabled.
-
-    Non-American Express cards are enabled on your Worldpay account by default.  If you want to accept American Express cards, you must also contact:
-      - the Government Banking Service (GBS) if your Worldpay contract is through the GBS
-      - your Worldpay account manager if your Worldpay contract is not through the GBS
-
-      You must send your American Express merchant ID to GBS or Worldpay if you’re a non-Crown merchant.
-
-2. Make a test transaction on your live account. You can use [the GOV.UK Pay
-   API](/api_reference) to do this.
-
-    Make sure you use a live API key, not a test one. Use [payment links](/payment_links) if your service is not yet connected to GOV.UK Pay.
-
-    If the test transaction fails, check that you have entered the correct credentials into the [GOV.UK Pay admin tool](https://selfservice.payments.service.gov.uk/).
-
-3. Sign in to your [GOV.UK Pay
-   account](https://selfservice.payments.service.gov.uk/).
-
-4. Go to the [__Transactions__
-   page](https://selfservice.payments.service.gov.uk/transactions) and check
-   your test transaction is in the list of transactions.
-
-5. Select the test transaction and check if you can refund it.
-
-6. Delete the payments link you created to make the test payment.
-
-    The refund option may take up to 20 minutes to appear after submitting the transaction.
-
-    If the refund option does not appear, check you have [set up notifications correctly](#set-up-merchant-channel).
-
-If you are testing your integration with a [payment link](/payment_links), you
-should open it in a separate browser tab to avoid confusion.
+- __SHOPPER_REDIRECTED__
+- __SENT_FOR_AUTHORISATION__
+- __AUTHORISED__
+- __ERROR__
+- __CANCELLED__
+- __CAPTURED__
+- __CAPTURE_FAILED__
+- __SETTLED__
+- __SETTLED_BY_MERCHANT__
+- __SENT_FOR_REFUND__
+- __REFUNDED__
+- __REFUNDED_BY_MERCHANT__
+- __REFUSED__
+- __REFUSED_BY_BANK__
+- __REFUND_FAILED__
 
 ## Set up 3D Secure
 
-Ask your Worldpay account manager to configure your merchant code to enable 3D
-Secure for all payments.
-
-When this is available, sign in to your [GOV.UK Pay
-account](https://selfservice.payments.service.gov.uk/login).
-
-Go to __My services__ to select the live service you want to set up.
-Select __Settings__, then the __View__ link under __3D Secure__. Then select __On__.
+1. Sign in to your [GOV.UK Pay account settings](https://selfservice.payments.service.gov.uk/settings/) and check 3D Secure is turned on.
+2. Ask your Worldpay account manager to enable 3D Secure for your merchant code.
```

Comment:
They should do these steps the other way round (otherwise we’ll try to send please-give-me-3DS requests to an account that isn’t expecting them).